includes/discovery/ports/edgeosolt.inc.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+unset($port_stats);
+
+$sfp_stats = snmpwalk_group($device, 'ubntSfps', 'UBNT-UFIBER-MIB', 1, $sfp_stats);
+
+$index = 0;
+for ($index = 0; $index < count($sfp_stats); $index++) {
```

Comment:
Can you change this to use foreach instead?

```php
foreach ($sfp_stats as $index => $stats) {
    $port_stats[$index + 1]['ifDescr'] = $stats['ubntSfpName'];
...
```

Reply:
> 

When using table instead of entry, nothing is detected.